### DISCUSSION TITLE
why it just have English???

### DISCUSSION THREAD

**Discussion by andy1145141414** [community/community#133816] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

help,I can't find where can change to Chinese!

**Comment by SohanKafle** [comment]
Hi [andy1145141414](https://github.com/andy1145141414),

If you're looking to change the language to Chinese on GitHub, unfortunately, GitHub's interface is primarily in English. However, here are some options and workarounds you can consider:

**1. Browser Translation:**
Use browser translation features. Most modern browsers like Chrome have built-in translation tools. Right-click on the page and select "Translate to 中文 (Chinese)".
**2. Localization Extensions:**
Use browser extensions that can translate web pages or specific sections into Chinese.
**3. Third-party Tools:**
Use third-party translation tools like Google Translate or DeepL to translate the content.
**4. GitHub Issues and Discussions:**
You can also look for or create issues/discussions in the [GitHub Community](https://github.com/community) or repositories to request support for additional languages.

If you need further assistance, feel free to ask!
